tests/test_cli.py

```
@@ -0,0 +1,57 @@
+import subprocess
+import sys
+
+
+def test_detect_shorts_cli(tmp_path):
+    circuit_file = tmp_path / "test_circuit.circuijt"
+    circuit_file.write_text("""
+    R R1
+    (GND) -- R1 -- (GND)
+    """)
+
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)